jobs/resource_job.go

```diff
@@ -286,6 +286,7 @@ type JobSettings struct {
 	Queue                *Queue                        `json:"queue,omitempty"`
 	RunAs                *JobRunAs                     `json:"run_as,omitempty"`
 	Health               *JobHealth                    `json:"health,omitempty"`
+	Parameters           []JobParameterDefinition      `json:"parameters,omitempty" tf:"alias:parameter"`
```

Comment:
we may need to stop using aliases as they may prevent from porting to Go SDK

@mgyucht WDYT?

Reply:
We should still be able to use the Go SDK as any tags/convenience methods here can be implemented in the closure passed to `common.StructToSchema`. E.g. `alias(old, new string)` would do `schema[new] = schema[old]; delete(old, schema)`.

Reply:
For the in-between period, we can define helper methods. Long-term, these attributes will need to be included in the OpenAPI spec itself.